nixos/modules/services/desktops/gnome3/gnome-keyring.nix

```diff
@@ -12,13 +16,11 @@ with lib;
 
     services.gnome3.gnome-keyring = {
 
-      enable = mkOption {
-        type = types.bool;
-        default = false;
+      enable = mkEnableOption "gnome-keyring";
+
+      enableService = mkEnableOption "gnome-keyring-daemon" // {
         description = ''
-          Whether to enable GNOME Keyring daemon, a service designed to
-          take care of the user's security credentials,
-          such as user names and passwords.
+          Whether to run a systemd service which runs the keyring daemon.
```

Comment:
How does gnome handle this usually? dbus activation?

Reply:
DBus activation is limited according to https://wiki.gnome.org/Projects/GnomeKeyring/RunningDaemon. PAM module is the preferred method. Maybe check https://github.com/NixOS/nixpkgs/pull/30686

Reply:
Arch wiki is also useful https://wiki.archlinux.org/index.php/GNOME/Keyring#Using_the_keyring_outside_GNOME